index.js

```diff
@@ -1,7 +1,9 @@
 const SlashtagsURL = require('@synonymdev/slashtags-url')
 const Noise = require('noise-handshake')
 const edCurve = require('noise-curve-ed')
-const fetch = require('node-fetch')
+const fetch = require('./fetch/fetch.js')
+const b4a = require('b4a')
+const { TextDecoder } = require('util')
```

Comment:
This also needs a mapping identical to `fetch.js`:

```suggestion
const TextDecoder = require('./textdecoder/textdecoder.js')
```

`./textdecoder/textdecoder.js`
```
const { TextDecoder } = require('util')
module.exports = TextDecoder
```

`./textdecoder/textdecoder-browser.js`
```
module.exports = globalThis.TextDecoder
```

Reply:
Right, missed that, thanks for catching